src/canvas/drawing_utils.rs

```diff
@@ -329,54 +120,4 @@ mod test {
         ));
         assert!(over_timer.is_none());
     }
-
-    #[test]
-    fn test_width_calculation() {
-        // TODO: Implement width calculation test; can reuse old ones as basis
-    }
-
-    #[test]
-    fn test_zero_width() {
-        assert_eq!(
-            get_column_widths(
-                0,
-                &[Some(1), None, None],
-                &[None, Some(1), Some(2)],
-                &[None, Some(0.125), Some(0.5)],
-                &[None, Some(10), Some(10)],
-                true
-            ),
-            vec![],
-        );
-    }
-
-    #[test]
-    fn test_two_width() {
-        assert_eq!(
-            get_column_widths(
-                2,
-                &[Some(1), None, None],
-                &[None, Some(1), Some(2)],
-                &[None, Some(0.125), Some(0.5)],
-                &[None, Some(10), Some(10)],
-                true
-            ),
-            vec![],
-        );
-    }
-
-    #[test]
-    fn test_non_zero_width() {
-        assert_eq!(
-            get_column_widths(
-                16,
-                &[Some(1), None, None],
-                &[None, Some(1), Some(2)],
-                &[None, Some(0.125), Some(0.5)],
-                &[None, Some(10), Some(10)],
-                true
-            ),
-            vec![2, 2, 7],
-        );
-    }
```

Comment:
Tests removed since this file no longer handles this logic. Tests were all moved along to `table_state.rs`.